docs/source/app_developers_guide/adding_a_node.rst

```diff
@@ -0,0 +1,443 @@
+   IntegerKey, XO, and Validator Registry transaction families. Run this
+   command:
+
+     .. code-block:: console
+
+        user@client-1$ sawset proposal create \
+        sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0"},{"family":"sawtooth_settings", "version":"1.0"},{"family":"xo", "version":"1.0"},{"family":"sawtooth_validator_registry", "version":"1.0"}]'
+
+#. Create the ``validator.toml`` configuration file on the first validator node.
+   Use the following command to copy the example configuration file to the
+   config directory, ``/etc/sawtooth``.
+
+   .. code-block:: console
+
+      user@client-1$ sudo cp /etc/sawtooth/validator.toml.example /etc/sawtooth/validator.toml
+
+   .. note::
+
+      This environment uses the default location and name for this configuration
+      file. For more information, see
+      :doc:`../sysadmin_guide/configuring_sawtooth/validator_configuration_file`.
+
+#. Edit ``/etc/sawtooth/validator.toml`` to change the following settings:
+
+   a. Set the ``bind`` directive to the network and component values for this
+      node.
+
+      * ``network`` specifies the validator-to-validator connection (by default,
+        ``127.0.0.1:8800``)
+
+      * ``component`` specifies the port on which the validator listens for
+        communication from other Sawtooth components (the REST API and
+        transaction processors)
+
+      For this application development environment, use the default settings:
+
+        .. code-block:: none
+
+           bind = [
+             "network:tcp://127.0.0.1:8800",
+             "component:tcp://127.0.0.1:4004"
+           ]
+
+   #. Set ``endpoint`` to the publicly addressable IP and port of the
+      validator-to-validator connection. By default, ``endpoint`` is the same as
+      the bind ``network`` setting: ``tcp://127.0.0.1:8800``. You must change
+      the ``endpoint`` value if the validator is hosted behind a firewall. In
+      this case, the firewall must map (redirect) the publicly addressable
+      IP:port to the bind network IP:Port.
+
+      .. code-block:: none
+
+         endpoint = "tcp://external-ip-1:8800"
+
+      The validator will share the endpoint IP:port with other validators when
+      it is peering with them.
+
+   #. Comment out all other settings in this file unless your network requires a
+      non-default configuration.
+
+      For information on the validator configuration options, see "Validator
+      Configuration File" in the System Administration Guide. For more information
+      on these settings, run ``sawtooth-validator --help`` on a validator node.
+
+.. _install-second-val-ubuntu-label:
+
+Step 2: Install and Configure a Second Validator Node
+------------------------------------------------------
+
+#. Install and configure Sawtooth on the second node as described in
+   :doc:`ubuntu`, with one difference: **Do not** create a genesis block
```

Comment:
If you don't remove the sawset command above, you may also need to explain you don't do that here or explain up there that you are making a blockchain setting to global state that you only do once. (gets wordy which is why I'd rather not have it in if it's not necessary).